addons/godot-traits/godot-traits.gd

```diff
@@ -7,6 +7,7 @@ class_name GodotTraitsEditorPlugin
 ##
 
 static var _instance:GodotTraitsEditorPlugin
+const commandKey_RegenGTraitsScript = "gtraits/regenerate-gtraits-script"
```

Comment:
For consistency:
- private vars start with an `_`
- I only use (or try to) snake case for naming

So maybe something like `_palette_command_key_regen_script` ?

Reply:
Works for me. I saw that issue, and couldn't think of a better name.